src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -106,7 +107,7 @@ async Task TrackResourceSnapshotsAsync()
                 loadingTcs.TrySetResult();
             }
 
-            _ = Task.Run(async () =>
+            _resourceSubscriptionTask = Task.Run(async () =>
```

Comment:
Why not use the `_resourceSubscriptionCancellation.Token` as an arg here, and then avoid the await in the `DisposeAsync`?

Reply:
Pass it to Task.Run? The CTS is never canceled at this point, so it won't have any impact.